mock/example/pod-test.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: web-server
+spec:
+  # deploy this pod only on where csi-mock-driver exists
+  # This is because csi-mock-driver keeps all states in memory and the process that
+  # provisioned the PV needs to be the same process that's mounting it
+  affinity:
```

Comment:
as a future improvement, if we add topology support to the mock driver, then we don't need this in the pod. The pv provisioned will have a nodeaffinity on it, and the pod will always get scheduled to the correct node.